### PR TITLE
[MM-41573] - Remove FF for MM-39060 Resend Invite Interval

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR removes the ResendInviteEmailInterval feature flag and defaults reminder invite email to be sent after 48 hours

#### Ticket Link
[MM-41573](https://mattermost.atlassian.net/browse/MM-41573)

#### Release Note

```release-note
NONE

```

### CODE REVIEW

**Review by julmondragon — APPROVED** [review]
LGTM. Very useful info!

**Review by stevemudie — APPROVED** [review]
I'll test this post merge.

**Comment by AGMETEOR** [comment]
Hey @nickmisasi gentle ping here

File: jobs/resend_invitation_email/worker.go [hunk 1]
```diff
@@ -87,52 +85,13 @@ func (rseworker *ResendInvitationEmailWorker) JobChannel() chan<- model.Job {
...
-			return
-		}
-	}
-
-	if (elapsedTimeSinceSchedule > firstDuration) && (systemValue == nil || systemValue.Value == "0") {
-		rseworker.ResendEmails(job, firstDurationTelemetryValue)
-		rseworker.setNumResendEmailSent(job, "1")
-	} else if elapsedTimeSinceSchedule > secondDuration {
-		rseworker.ResendEmails(job, secondDurationTelemetryValue)
+func (rseworker *ResendInvitationEmailWorker) DoJob_48(job *model.Job) {
```
Comment (JulienTant) [inline jobs/resend_invitation_email/worker.go:91]: nit: we can merge `DoJob_48` into `DoJob`

File: jobs/resend_invitation_email/worker.go [hunk 2]
```diff
@@ -188,41 +140,20 @@ func (rseworker *ResendInvitationEmailWorker) removeAlreadyJoined(teamID string,
...
 
-	duration_72 := os.Getenv("MM_RESEND_INVITATION_EMAIL_JOB_DURATION_72")
-	DurationInMillis_72, parseError := strconv.ParseInt(duration_72, 10, 64)
-	if parseError != nil {
-		// default to 72 hours
-		DurationInMillis_72 = SeventyTwoHoursInMillis
-	}
-
-	return elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_48, DurationInMillis_72
+	return elapsedTimeSinceSchedule, DurationInMillis_48
```
Comment (JulienTant) [inline jobs/resend_invitation_email/worker.go:156]: nit: no need for the `_48`'s anymore

**Review by JulienTant — APPROVED** [review]
LGTM Overall, I think we can remove the references to `48` in some places to make the code more readable.

**Pull request merged**